specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -25,22 +25,19 @@
     }
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/anomalydetector/{ApiVersion}",
+    "hostTemplate": "{Endpoint}/anomalydetector",
     "useSchemePrefix": false,
     "parameters": [
       {
         "$ref": "#/parameters/Endpoint"
-      },
-      {
-        "$ref": "#/parameters/ApiVersion"
       }
     ]
   },
   "schemes": [
     "https"
   ],
   "paths": {
-    "/multivariate/detect-batch/{resultId}": {
+    "/{ApiVersion}/multivariate/detect-batch/{resultId}": {
```

Comment:
I will assume that that this is the pattern that we suggest for all path version based services? (e.g. not have the api version in the host template).

Reply:
We need this https://github.com/Azure/autorest/issues/4656 in order to suggest something different, right now this is the only way to make it work at all.